app/main.py

```diff
@@ -1,5 +1,5 @@
 class Animal:
-    alive = []
+    alive: list = []
```

Comment:
```suggestion
    alive: list[Animal] = []
```

Reply:
this syntaxis results in "Unresolved reference 'Animal' "

Reply:
You can use:
`from __future__ import annotations`